configs/yesterdays_surface_example.cfg

```diff
@@ -13,7 +13,10 @@
 # limitations under the License.
 
 [parameters]
-target_path=all.an
+client=mars
+partition_keys=
+    type
```

Comment:
Looking at this again, isn't it possible to download everything into one chunk? If so, why do we need a partition at all? If this isn't supported by the parser / dry-runs, then I think it's a bug (we can raise an issue for later). 

Further, `type` is a strange thing to partition on. I think `date` or `param` would be better.

Reply:
Good point! There is a `KeyError` [here ](https://github.com/google/weather-tools/blob/main/weather_dl/download_pipeline/parsers.py#L223) when `partition_keys` is not set

Reply:
Created #59 to track